src/commands/infra/generate.test.ts

```diff
@@ -52,6 +52,147 @@ beforeEach(() => {
   jest.restoreAllMocks();
 });
 
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
Did you mean to keep this?

Reply:
to clearly separate git related tests from other tests